python/example_script.py

```diff
@@ -0,0 +1,107 @@
+"""An example Python script showing IAI Style Guide format and argparse usage.
+
+This Python script implements a dummy function that takes input and output file
+paths. The main purpose of this script is to show how a Python file should look
+in order to be presentable and conform to the IAI Python Style Guide. As a
+bonus, the usage of the argparse library is demonstrated for use with command-
+line interface (CLI).
+
+    Typical usage example in shell CLI:
+
+    python example_script.py -i input.txt -o output.txt -m buzzfizz.json
+
+    Or simply:
+
+    python example_script.py -i input.txt -o output.txt
+
+:Author: Trond Linjordet
+"""
+
+import argparse
+import json
+
+
+def dummy_line_modification(line: str, i: int, mods: dict) -> str:
```

Comment:
Further specify dict keys and values using typing.Dict

Reply:
Done.